src/main/java/com/google/sps/data/Referendum.java

```diff
@@ -66,6 +72,7 @@ public static Referendum fromJSONObject(JSONObject obj) throws JSONException {
     String referendumDescription;
```

Comment:
can you remove the "referendum" prefix?
Or even better, can we just set the properties in the Referendum.builder() directly?

Reply:
We use a try/catch when reading the JSON object because the description may or may not exist in the JSON object. Happy to rename the variable, but setting directly in the builder() may cause runtime errors that we can avoid.